go/vt/servenv/buildinfo.go

```diff
@@ -31,6 +32,7 @@ var (
 	buildTime      = ""
 	buildGitRev    = ""
 	buildGitBranch = ""
+	buildNumberStr = ""
```

Comment:
Can you please rename this to "jenkinsBuildNumberStr"?

Same comment for other variables throughout the file.

I say that because I assume that the build number is specific to Jenkins and not a general concept?